app/views/ctc/questions/stimulus_received/edit.html.erb

```diff
@@ -8,10 +8,9 @@
   <div class="review-box spacing-below-60">
     <div class="review-box__title spacing-below-15">
       <h2><%= t("views.ctc.questions.stimulus_received.amount_received") %></h2>
-      <%= link_to t("general.edit").downcase, questions_stimulus_one_path, class: "review-box__edit" %>
+      <%= link_to t("general.edit").downcase, questions_stimulus_three_path, class: "review-box__edit" %>
     </div>
-    <p class="spacing-below-15"><%= t("views.ctc.questions.stimulus_received.eip_one") %>: <%= calculated_or_provided_dollar_amount(current_intake.eip1_amount_received) %></p>
-    <p class="spacing-below-0"><%= t("views.ctc.questions.stimulus_received.eip_two") %>: <%= calculated_or_provided_dollar_amount(current_intake.eip2_amount_received) %></p>
+    <p class="spacing-below-0"><%= t("views.ctc.questions.stimulus_received.eip_three") %>: <%= number_to_currency(current_intake.eip3_amount_received) %></p>
```

Comment:
should we round this number and use `calculated_or_provided_dollar_amount`? otherwise includes "$.00"